src/less/icon/base/icon.less

```diff
@@ -1,3 +1,5 @@
+@import "../../tokens/@{ds}/tokens.less";
+@import "../../mixins/icon/base/icon-mixins.less";
 @import "../../mixins/utility/utility-mixins.less";
 
```

Comment:
The icon override seems to be missing from here: `@confirmation-icon-color: @color-g4;` .

Was master merged/rebased into `14.0.0` and/or this branch?